modules/graphql_views/src/Plugin/GraphQL/Fields/View.php

```diff
@@ -67,6 +67,17 @@ public function resolveValues($value, array $args, ResolveInfo $info) {
       $executable = $view->getExecutable();
       $executable->setDisplay($definition['display']);
 
+      // Set view arguments (contextual filters) if required.
+      /* @see \Drupal\graphql_views\Plugin\Deriver\ViewDeriver::getContextualSets() */
+      foreach ($definition['contextual_sets'] as $contextual_set) {
+        if (isset($contextual_set['argument_entity_class']) && is_a($value, $contextual_set['argument_entity_class'])) {
+          $executable->setArguments([
+            $contextual_set['argument'] => $value->id(),
+          ]);
+          break;
```

Comment:
How about introducing a "context" input argument (similar to "filter"), so contextual arguments can be populated on root level too. And when executed in context of an entity, we use it to fill in the blanks.